hack/e2e-test-ocp.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+# e2e-test-ocp.sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export OC=$(which oc) # OpenShift CLI
+SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+ROOT_DIR="$SOURCE_DIR/.."
+
+# Empty value for OCP tests.
+export E2E_KIND_VERSION=""
+# shellcheck source=hack/e2e-common-ocp.sh
+source "${SOURCE_DIR}/e2e-common.sh"
+
+function wait_for_cert_manager_ready() {
+    echo "Waiting for cert-manager components to be ready..."
+    local timeout=300
+    local interval=10
+    local elapsed=0
+
+    while true; do
```

Comment:
I think we could use https://docs.openshift.com/container-platform/4.17/cli_reference/openshift_cli/developer-cli-commands.html#oc-wait
instead of this script.